05/app.js

```diff
@@ -0,0 +1,47 @@
+const formEl = document.querySelector('form');
+const ulEl = document.querySelector('ul');
+formEl.addEventListener('submit', validateData);
+
+function validateData(e) {
+    const errors = [];
+    const firstName = e.target.elements.firstName.value;
+    const lastName = e.target.elements.lastName.value;
+    const street = e.target.elements.street.value;
+    const houseNumber = Number(e.target.elements.houseNumber.value);
+    const zip = e.target.elements.zip.value;
+    const city = e.target.elements.city.value;
+    const voivodeship = e.target.elements.voivodeship.value;
+    if (firstName.length < 2) {
+        errors.push('Błędnie podano Imię');
+    }
+    if (lastName.length < 2) {
+        errors.push('Błędnie podano Nazwisko');
+    }
+    if (street.length < 2) {
+        errors.push('Błędnie podano Nazwę ulicy');
+    }
+    if (houseNumber <= 0) {
+        errors.push('Błędnie podano Numer budynku');
+    }
+    if (zip.length !== 6 || zip.match(/[0-9][0-9]-[0-9][0-9][0-9]/) === null) {
```

Comment:
Lapiej byłoby to zapisać jako `[0-9]{2}-[0-9]{3}` :)

Reply:
Ok. Zmienione